### ISSUE TITLE
Clarify an answer for question 10

### ISSUE THREAD

**Issue by lauripiisang** [lydiahallie/javascript-questions#57]
> 10. What happens when we do this?

> C: `undefined`

Undefined is a value, not an event, so it should be clarified what the context for it is.

Perhaps something like 

> C: `bark` becomes `undefined` 

?

**Comment by A1rPun** [comment]
The answers to the question, like `SyntaxError`, imply that the syntax is getting parsed so the outcome isn't specifically for some variables to resolve.

The script from the question will output `"dog"` as last statement and that isn't one of the answers too..
 
##### Example:
What will happen?
```js
var a = 1;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
```
 - A: Nothing, this is totally fine!
 - B: `SyntaxError` 
 - C: `undefined`
 - D: `1`

**Comment by A1rPun** [comment]
The answers to the question, like `SyntaxError`, imply that the syntax is getting parsed so the outcome isn't specifically for some variables to resolve but I do agree that `undefined` is like an alias for *Nothing, this is totally fine!* because the syntax is already parsed at that stage. 

The script from the question will output `"dog"` as last statement and that isn't one of the answers, yet. So maybe reword it something like this:
 
##### Example:
What happens when we do this?
```js
var a = 1;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
```
 - A: The script will output `undefined`
 - B: The script will output `1`
 - C: `SyntaxError`

**Comment by lydiahallie** [comment]
I added a more reasonable choice, thanks 👍
